### PR TITLE
Multi sentry

### PR DESCRIPTION
**Pull request by AskAlexSharov**

### CODE REVIEW

**Review by AlexeyAkhunov — APPROVED** [review]

**Comment by AlexeyAkhunov** [comment]
Looks good to me in the first approximation. This to think about next:
1) instead of sending all messages to `sentries[0]` perhaps try to randomise? But for now (perhaps for a long time) approach of "leading sentry" will probably work well
2) We will need to introduce TLS encryption to ensure data integrity. But I think instead of mutual authentication, perhaps sentries should just authenticate the downloader?

**Comment by AskAlexSharov** [comment]
1. Thinking about it... grpc has built-in client-side load balancing, but as I understand - can’t get both and “load balancing” and “send to all servers”. Will try more tomorrow. 
2. Ok. Will decide tomorrow. I need to read - if TLS is compatible with DNS discovery in gRPC.

File: cmd/headers/commands/root.go [hunk 1]
```diff
@@ -16,11 +16,12 @@ import (
 )
 
 var (
-	sentryAddr    string // Address of the sentry <host>:<port>
-	coreAddr      string // Address of the core <host>:<port>
-	chaindata     string // Path to chaindata
-	database      string // Type of database (lmdb or mdbx)
-	mapSizeStr    string // Map size for LMDB
+	sentryAddr    string   // Address of the sentry <host>:<port>
```
Comment (AlexeyAkhunov) [inline cmd/headers/commands/root.go:19]: We don't need that one anymore?

File: cmd/headers/commands/root.go [hunk 2]
```diff
@@ -16,11 +16,12 @@ import (
...
+	sentryAddrs   []string // Address of the sentry <host>:<port>
+	coreAddr      string   // Address of the core <host>:<port>
```
Comment (AlexeyAkhunov) [inline cmd/headers/commands/root.go:21]: I forgot to get rid of this one, I think

**Pull request merged**